### DISCUSSION TITLE
Timeline of several recipes to coordinate cook times of large meals

### DISCUSSION THREAD

**Discussion by Majawat** [mealie-recipes/mealie#3545] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

If I'm planning a large gathering (Thanksgiving), it would be nice to be able to have a timeline view of when I need to start/finish cooking a dish to make sure things are on the table at specified times.

### Please provide a concise description of the feature that would resolve your issue.

During Meal Planning, being able to put in what time a certain dish needs to be completed, and using the Total Time from the recipe to back track when to start a dish

I do this in a spreadsheet currently, so this may help explain my goal:
![image](https://github.com/mealie-recipes/mealie/assets/12058855/add37c88-a844-40fd-9b31-09664c463d57)

### Please consider and list out some caveats or tradeoffs made in your design decision

As is, this wouldn't factor in specific steps so it would be very general. Start cooking X dish at 9am, etc.

Bonus feature would be that each step is associated to a specified amount of time. "Cook turkey 5 hours" for one step, "rest Turkey for 30 minutes" for another step.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by MatzeG1t** [comment]
related to  #5094 #4949 #1573
